plugins/main/public/components/overview/mitre/framework/mitre.tsx

```diff
@@ -0,0 +1,216 @@
+  });
+
+  const { searchBarProps } = useSearchBar({
+    indexPattern: dataSource?.indexPattern as IndexPattern,
+    filters,
+    setFilters: setFilters,
+  });
+  const [mitreState, setMitreState] = useState<tMitreState>({
+    tacticsObject: {},
+    selectedTactics: {},
+  });
+
+  const [filterParams, setFilterParams] = useState<tFilterParams>({
+    filters: fetchFilters,
+    query: searchBarProps?.query,
+    time: {
+      from: searchBarProps?.dateRangeFrom,
+      to: searchBarProps?.dateRangeTo,
+    },
+  });
+  const [indexPattern, setIndexPattern] = useState<any>(); //Todo: Add correct type
+  const [isLoading, setIsLoading] = useState(true);
+
+  const initialize = async () => {
+    setIndexPattern(dataSource?.indexPattern);
+    let filterParams = {
+      filters: fetchFilters, // pass the fetchFilters to use it as initial filters in the technique flyout
+      query: searchBarProps?.query,
+      time: {
+        from: searchBarProps?.dateRangeFrom,
+        to: searchBarProps?.dateRangeTo,
+      },
+    };
+    setFilterParams(filterParams);
+    setIsLoading(true);
+    await buildTacticsObject();
+  };
+
+  useEffect(() => {
+    if (isDataSourceLoading || !dataSource) return;
+    initialize();
+  }, [
+    isDataSourceLoading,
+    dataSource,
+    searchBarProps.query,
+    searchBarProps.dateRangeFrom,
+    searchBarProps.dateRangeTo,
+    JSON.stringify(filters),
+  ]);
+
+  const buildTacticsObject = async () => {
+    try {
+      const data = await WzRequest.apiReq('GET', '/mitre/tactics', {});
+      const result = (((data || {}).data || {}).data || {}).affected_items;
+      const tacticsObject = {};
+      result &&
+        result.forEach((item) => {
+          tacticsObject[item.name] = item;
+        });
+      setMitreState({ ...mitreState, tacticsObject });
+      setIsLoading(false);
+    } catch (error) {
+      setMitreState({ ...mitreState });
+      setIsLoading(false);
+      const options = {
+        context: `${Mitre.name}.buildTacticsObject`,
+        level: UI_LOGGER_LEVELS.ERROR,
+        severity: UI_ERROR_SEVERITIES.BUSINESS,
+        store: true,
+        display: true,
+        error: {
+          error: error,
+          message: error.message || error,
+          title: `Mitre data could not be fetched`,
+        },
+      };
+      getErrorOrchestrator().handleError(options);
+    }
+  };
+
+  const onChangeSelectedTactics = (selectedTactics) => {
+    setMitreState({ ...mitreState, selectedTactics });
+  };
+
+  const flexGroupStyle = {
+    margin: '0 8px',
+  };
+
+  return (
+    <div>
+      <I18nProvider>
+        <EuiFlexGroup style={flexGroupStyle}>
+          <EuiFlexItem>
+            {isDataSourceLoading && !dataSource ? (
+              <LoadingSpinner />
+            ) : (
+              <div className="wz-discover hide-filter-control wz-search-bar">
+                <SearchBar
+                  appName="vulnerability-detector-searchbar"
```

Comment:
I think this should be `mitre-attack-searchbar`

Reply:
Solved in da292a701c672ad91a2fb285a1a72fbfbaa178a7